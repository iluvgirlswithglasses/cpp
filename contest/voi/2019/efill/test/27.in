38
-80058050 50875262
-80057964 -50311910
-79938038 -79930480
-79997216 -79746232
-64790710 -80057798
77631188 80053216
-68923350 -80050084
80036880 78719122
-80023914 -79384806
-65033198 80055782
-80031992 -79061062
79834988 -79973230
79953098 79774048
-79240146 -80016920
-13337544 -80058210
-72669426 80054746
-80002886 79385846
80058174 -23055002
-76702040 -80026052
80057766 -32024054
79871108 -79969988
-77160054 80051238
78177716 -80051430
80057944 -12916440
80042648 -79086040
-80056126 73290432
-79766056 80005368
-26719510 80057860
80041446 76286808
79999690 79356172
80054994 -74329474
80052380 -76109118
68137498 80058044
79860372 80024826
80004866 -79336698
-80056566 -60954624
80056020 63849304
75268156 -80053838
21129122 50875348 21129208 -50311824
