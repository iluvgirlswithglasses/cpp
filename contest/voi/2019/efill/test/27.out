719869953.34
